contracts/libraries/Token.sol

```diff
@@ -61,9 +75,7 @@ library Token {
     } else if (_info.erc == Standard.ERC721) {
       // bytes4(keccak256("transferFrom(address,address,uint256)"))
       (_success, ) = _token.call(abi.encodeWithSelector(0x23b872dd, _from, _to, _info.id));
-    } else {
-      revert("Token: unsupported token standard");
-    }
+    } else revert ErrUnsupportedStandard();
 
     if (!_success) {
```

Comment:
Please change the below `string(abi.encodePacked)` into custom error.

https://github.com/axieinfinity/ronin-dpos-contracts/pull/243/files#diff-44e5707d9a5ca48b8e94d327904aa677cce540d6c8d7f465a69e185b7d7db3d2R81-R94